md-dao/src/main/java/com/wipro/ats/bdre/md/triggers/MetadataPersistenceTrigger.java

```diff
@@ -30,7 +31,7 @@
 public class MetadataPersistenceTrigger implements PreUpdateEventListener, PreInsertEventListener, PostUpdateEventListener, PostInsertEventListener {
 
     private static final Logger LOGGER = Logger.getLogger(MetadataPersistenceTrigger.class);
-    private static HashMap<Integer,Integer> processTypeMap=new HashMap<>();
+    private static Map<Integer,Integer> processTypeMap=new HashMap<>();
     //TODO:We have to populate this from DB
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1135)